src/App.tsx

```diff
@@ -35,13 +36,28 @@ const App: React.FC = () => {
     return ready;
   }, []);
 
+  const onConnectButton = React.useCallback(async (onboard: OnboardAPI | null, provider): Promise<boolean> => {
+    if (!provider) {
+      const walletSelected = await onboard?.walletSelect();
+      if (!walletSelected) return false;
```

Comment:
Let's add braces here and in the other `if` statement, just for consistency!

Reply:
Fixed in c95b264f4de3242498156353b1ac1179fe889b42